### PR TITLE
fix: CPE does not start UI Adaptation for ui5 versions lower than 1.120 for Adaptation Projects

### LINKED ISSUE #1929: BUG - CPE does not start UI Adaptation for ui5 versions lower than 1.120 for Adaptation Projects
### Related Feature
Feature request: #1874

### Description
CPE breaks for Adaptation Projects with the ui5 version lower than 1.120.

This import is causing the error in `packages/preview-middleware-client/src/adp/init.ts`: 

```ts
import UI5ElementRegistry from 'sap/ui/core/ElementRegistry';
```

### Steps to Reproduce
1. Start and ADP with ui5 version 1.108.31 for example
2. Open dev tools and look at the console
3. See the error (screenshot below)

### Expected results
UI Adaptation is started by RTA

### Actual results
RTA does not start.

### Screenshots
If applicable, add screenshots to help explain the problem.

Screenshot 2024-05-15 at 13 48 05

Screenshot 2024-05-15 at 13 49 30

#### Version/Components/Environment	
Add any other context about the problem here
OS:                      
- [ ] Mac OS
- [ ] Windows
- [ ] Other

### Problem
{describe the problem} 

### Fix
{describe the fix} 

### Why was it missed
{Some explanation why this issue might have been missed during normal development/testing cycle}  

### How can we avoid this
{if we don’t want to see this type of issues anymore what we should do to prevent}

### PR DESCRIPTION
**Pull request by GDamyanov**
fixes #1929 

I successfully tested with UI5 Versions lower and bigger than 1.120

### CODE REVIEW

**Review by nikmace — COMMENTED** [review]
Changes look good
Test coverage good
Did not test manually

File: packages/preview-middleware-client/src/adp/init.ts [hunk 1]
```diff
@@ -46,7 +44,7 @@ export default async function (rta: RuntimeAuthoring) {
         }
     );
 
-    const syncViewsIds = getAllSyncViewsIds();
+    const syncViewsIds = await getAllSyncViewsIds(minor);
```
Comment (voicis) [inline packages/preview-middleware-client/src/adp/init.ts:47]: This seems to be executed before the app is completely loaded which means that maybe some of the views are not loaded. But this is not related to this particular fix.

**Review by voicis — APPROVED** [review]
- Code changes look good
- Test coverage is good
- Tested locally with UI5 `1.20.14` and `1.123.1`

**Review by nikmace — APPROVED** [review]
Changes look good
Test coverage good
Did not test manually
